docs/src/starting.md

```diff
@@ -43,11 +43,11 @@ julia> x = rand(d, 100)
 
 You can easily obtain the `pdf`, `cdf`, `quantile`, and many other functions for a distribution. For instance, the median (50th percentile) and the 95th percentile for the standard-normal distribution are given by:
 
-```julia-repl
+```jldoctest getting-started
```

Comment:
Maybe we could add a doctest filter to avoid that tests start to fail due to differences in the hardware architecture or Julia version (leading to slightly different last digits)?